tests/foreman/destructive/test_clone.py

```diff
@@ -27,7 +27,9 @@
 
 
 @pytest.mark.parametrize(
-    "sat_ready_rhel", [7, 8] if settings.server.version.rhel_version < 8 else [8], indirect=True
+    "sat_ready_rhel",
+    [7, 8] if int(float(settings.server.version.rhel_version)) < 8 else [8],
```

Comment:
Even better fix would be
```suggestion
    [7, 8] if get_sat_version().major < 8 else [8],
```

Reply:
you mean `get_sat_rhel_version().major`

Reply:
```
In [10]: from robottelo.hosts import get_sat_rhel_version

In [11]: get_sat_rhel_version()
Out[11]: <Version('8.7')>

In [12]: get_sat_rhel_version().major
Out[12]: 8

```